index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My first Portfolio</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.4/css/all.css" integrity="sha384-DyZ88mC6Up2uqS4h/KRgHuoeGwBcD4Ng9SiP4dIRy0EXTlnuz47vAwmeGwVChigm" crossorigin="anonymous">
+</head>
+<body>
+    <nav class="toolbar">
+        <h3>Moise Mulungu</h3>
+        <div class="bars">
+            <i class="fas fa-bars"></i>
+        </div>
+    </nav>
```

Comment:
Hi, @moise-mulungu ✋
- [ ] We have to use a `header` tag for the toolbar, the container for this section is a `header` tag. Please make sure to add a `header` tag on this section.  

- [ ] I think we can add a link (`a` tag) to the logo in `h3`. 

- [ ] Also, we must stick to [the Figma template design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471) as much as possible. Please add the left padding to the logo `h3`.

- [ ] I think we are missing the hamburger icon here. There are a few websites for icons, you can check [fontawesome](https://fontawesome.com/v5.15/icons/bars?style=solid). You can also check [this link](https://www.w3schools.com/icons/) for more information about adding icons 😊 .